config/clusters/nasa-ghg/common.values.yaml

```diff
@@ -70,6 +70,50 @@ basehub:
             resource_allocation: &profile_options_resource_allocation
               display_name: Resource Allocation
               choices:
+                # The `cpu_` options are added by hand, temporarily,
+                # to help evaluate ideal resource options for an upcoming workshop.
+                # See: https://github.com/NASA-IMPACT/veda-jupyterhub/issues/32
+                # Once we settle on the ideal resource allocation for the workshop, we should:
+                #  - Create a new profile for the workshop, with just the resource option needed
+                #  - Pick a more appropriate (compute-optimized) node type and add to eksctl
+                #  - Remove these options from the global resource select options
+                #  - Remove this comment.
+                cpu_8_16:
+                  display_name: 8 CPU, 16GB RAM
+                  kubespawner_override:
+                    mem_guarantee: 14.87Gi
```

Comment:
JupyterHub wants these to be 'G' rather than 'Gi'. Hence the issue. Can you change mem_guarantees to G?

Additionally, I'd love for you to also look at supporting the real 'i' suffixes in https://github.com/jupyterhub/jupyterhub/blob/fa538cfc65b199dd2947529e349abb9aa31fb9a3/jupyterhub/traitlets.py#L49 if you'd like, @batpad :)

Reply:
Thanks - sorry I should have double checked the units there - I made another PR to fix this: https://github.com/2i2c-org/infrastructure/pull/4048